zerver/webhooks/newrelic/view.py

```diff
@@ -13,10 +13,11 @@
 
 @api_key_only_webhook_view("NewRelic")
 @has_request_variables
-def api_newrelic_webhook(request, user_profile, stream=REQ(default='newrelic'),
-                         alert=REQ(validator=check_dict([]), default=None),
-                         deployment=REQ(validator=check_dict([]), default=None)):
-    # type: (HttpRequest, UserProfile, Text, Optional[Dict[str, Any]], Optional[Dict[str, Any]]) -> HttpResponse
+def api_newrelic_webhook(request: HttpRequest, user_profile: UserProfile,
+                         stream: Text=REQ(default='newrelic'),
+                         alert: Optional[Dict[str, Any]]=REQ(validator=check_dict([]), default=None),
+                         deployment: Optional[Dict[str, Any]]=REQ(validator=check_dict([]), default=None)
+                         )-> HttpResponse:
```

Comment:
Maybe align this last line with 0-indent like we've done in earlier PRs?

Reply:
By 0-indent, do you mean aligned with the initial definition? Because the linter seems to dislike that the closing bracket does not match the visual indentation D:

Reply:
Never mind @picapi, it looks good now :stuck_out_tongue: